docs/content/ingestion/batch-delta-ingestion.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: doc_page
+---
+
```

Comment:
How do you feel about giving this section a title 'Updating Existing Data'?

And then we can explain updates in Druid are done by reindexing or rebuilding segments, and by delta ingestion. We can also mention that lookups provide a way to do updates on a small dimension table.